.storybook/recipes/CalendarCard/CalendarCard.module.css

```diff
@@ -0,0 +1,59 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #CALENDAR CARD
+\*------------------------------------*/
+
+/**
+ * Card recipe for displaying calendar data
+ */
+.calendar-card {
+  width: 16rem;
+  justify-content: space-between;
+  gap: var(--eds-size-1);
+  background-color: var(--eds-theme-color-background-neutral-default-hover);
+}
+
+.calendar-card--brand {
+  border-left: var(--eds-border-width-lg) solid
+    var(--eds-theme-color-border-brand-primary-strong);
+}
+
+.calendar-card--revise {
+  border-left: var(--eds-border-width-lg) solid
+    var(--eds-theme-color-border-grade-revise-strong);
+}
+
+.calendar-card--success {
+  border-left: var(--eds-border-width-lg) solid
+    var(--eds-theme-color-border-utility-success-strong);
+}
+
+.calendar-card__title {
+  @mixin eds-typography-preset-007-bold;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: var(--eds-size-1);
+}
+
+.calendar-card__title--icon {
+  min-width: 1.2rem;
+}
+
+.calendar-card__dates {
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  gap: var(--eds-size-1);
+}
+
+.calendar-card__dates--icon {
```

Comment:
Let's call this `dates-icon` and not `dates--icon`. We don't use this syntax since it looks like a modifier when it should just be a child. So we always use a `[child]-[part-of-child]` if we need to be specific generally